tests/dedupe_unit_test.py

```diff
@@ -24,7 +24,7 @@ def setUp(self):
     def check_nb_duplicates(self, expected_number_of_duplicates):
         print("checking duplicates...")
         retries = 0
-        for i in range(0, 3):
+        for i in range(0, 18:
```

Comment:
Missing `)`?

Reply:
crap, used the web ui. I knew I had better just left it as is.